.vscode/settings.json

```diff
@@ -1,11 +1,13 @@
 // Place your settings in this file to overwrite default and user settings.
 {
-    "typescript.validate.enable": false,
-    "typescript.format.enable": false,
     "flow.useNPMPackagedFlow": true,
     "flow.enabled": true,
+    "eslint.autoFixOnSave": true,
     "javascript.validate.enable": false,
```

Comment:
Enabling this to true breaks hell loose, with TSC trying to check .js files ignoring tsconfig.json